controls/powershell.rb

```diff
@@ -2,6 +2,7 @@
 
 title 'Windows PowerShell'
 
+## FIXME! CIS says disabled, fireeye enabled https://www.fireeye.com/blog/threat-research/2016/02/greater_visibilityt.html
```

Comment:
If we cannot decide, lets create an inspec attribute to enable or disable that feature. I think we should follow the CIS default.